util/http/http_handler.h

```diff
@@ -80,6 +80,12 @@ class HttpListenerBase : public ListenerInterface {
     enable_metrics_ = true;
   }
 
+  // Enforce authentication to this HTTP server with the user "user"
+  // and this password.
+  void SetPassword(std::string password) {
```

Comment:
Correct me if I'm wrong, but this is dead code, right? (i.e. nothing calls `SetPassword()`)
Where do you intend to call this method?

Reply:
From `facade::Connection::HandleRequests`.

Reply:
Ok, please add `override` then :)

Reply:
But I now added a demonstration to `http_main`

Reply:
It's not virtual though?

Reply:
Now I understand, thanks!